app/imports/components/Users/AdminPanel/UsersManagement/UsersList/usersList.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Avatar from 'material-ui/Avatar';
+import Subheader from 'material-ui/Subheader';
+import { List, ListItem, makeSelectable } from 'material-ui/List';
+import { PropTypes } from 'prop-types';
+import SetAdmin from './setAdmin';
+
+
+export default class UsersList extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.SelectableList = makeSelectable(List); // @TODO Do it like in tutorial
+    }
+
+
+    render() {
+        return (
+            <div>
+                <this.SelectableList>
+                    <Subheader>Users</Subheader>
+                    {this.props.users.map(user => (
+                        <ListItem
+                            value={1}
```

Comment:
I do not know too. It is not working without this. I don't know whole documentation for every component, you can check by yourself.